raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -10,7 +10,7 @@ contract NettingChannelContract {
     event ChannelClosed(address closing_address, uint block_number);
     event TransferUpdated(address node_address, uint block_number);
     event ChannelSettled(uint block_number);
-    event ChannelSecretRevealed(bytes32 secret);
+    event ChannelSecretRevealed(bytes32 secret, address receiver_address);
```

Comment:
Ah got it: our current `unlock` below is a special case that only the `receiver` of the hash locked transfer can call. Thanks!